src/Symfony/Component/Lock/Key.php

```diff
@@ -77,23 +77,31 @@ public function getState($stateKey)
      */
     public function reduceLifetime($ttl)
     {
-        $newExpiringDate = \DateTimeImmutable::createFromFormat('U.u', (string) (microtime(true) + $ttl));
+        $newTime = microtime(true) + $ttl;
 
-        if (null === $this->expiringDate || $newExpiringDate < $this->expiringDate) {
```

Comment:
Switch to float comparison instead of datetime, due to bug in PHP => https://3v4l.org/bSt3d